GPflowOpt/optim.py

```diff
@@ -175,21 +161,39 @@ def _optimize(self, objective):
                               nfev=points.shape[0],
                               message="OK")
 
+    def set_initial(self, initial):
```

Comment:
I assume the domain.setter is overrided above because you don't want to have the warning printed on every domain set?